lib/base-controller.js

```diff
@@ -59,11 +59,13 @@ BaseController.prototype.getErrorStep = function getErrorStep(err, req) {
 
 BaseController.prototype.locals = function controllerLocals(req, res) {
   var locals = Controller.prototype.locals.apply(this, arguments);
+  var meta = (this.options || {}).meta;
+
   return _.extend({}, locals, {
     baseUrl: req.baseUrl,
     nextPage: this.getNextStep(req, res),
     errorLength: getErrorLength.apply(this, arguments)
-  });
+  }, meta || {});
```

Comment:
if `this.options.meta` is `undefined` then we are extending `locals` with `undefined`;

It would probably be better placed on line 62 though - `var meta = (this.options || {}).meta || {};`